app/commands/todo/abandon.rb

```diff
@@ -7,12 +7,17 @@ module Abandon
         class Command
           attr_reader :payload, :aggregate_id
 
+          def self.build(**args)
```

Comment:
We preferred to keep them all inline to make it easier for people to read the code in one place. Mainly because this repo is aimed at being educational.